pom.xml

```diff
@@ -22,7 +22,7 @@
 
   <groupId>io.cdap.plugin</groupId>
   <artifactId>condition-plugins</artifactId>
-  <version>1.4.0</version>
+  <version>1.4</version>
```

Comment:
this should be 1.4.0 (we use major, minor and patch version pattern)

Reply:
done